src/Microsoft.Health.Fhir.Core/Features/Validation/NarrativeValidator.cs

```diff
@@ -14,7 +15,7 @@
 
 namespace Microsoft.Health.Fhir.Core.Features.Validation.Narratives
 {
-    public class NarrativeValidator : NoopPropertyValidator
+    public class NarrativeValidator<T, TProperty> : NoopPropertyValidator<T, TProperty>
```

Comment:
>T, TProperty [](http://example.com/codeflow?start=36&length=12)

since we pass <ResourceElement,ResourceElement>
can it be:

```
NarrativeValidator<T> : NoopPropertyValidator<T, T>
```
?